release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -515,6 +515,78 @@ For more information, see xref:../security/tls-security-profiles.adoc#tls-profil
 [id="ocp-release-notes-storage_{context}"]
 === Storage
 
+[id="ocp-release-notes-storage-azure-file-cross-sub-support_{context}"]
+==== Azure File cross-subscription support is generally available
+Cross-subscription support allows you to have an {product-title} cluster in one Azure subscription and mount your Azure file share in another Azure subscription using the Azure File Container Storage Interface (CSI) driver.
```

Comment:
within the same tenant, OCP and the associated storage can be in different subcriptions but in the same Azure tenant

Reply:
Fixed.